education.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My education details</title>
+	<meta charset="utf-8">
+	<meta name="Description" content=This page show my education details">
+	<meta name="keywords" content="nazish, ABSS Institute of Technology, SBSS College, Education">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+	<header>
+		<h1>Nazish Ahmad</h1>
+	    <p>Hi, I'm Nazish. A computer science engineering student who love to solve problem.</p>
+	</header>
+	<main>	
+	    <h2>My Education Details.</h2>
+	    <table>
+	    	<thead>
+	    		<tr>
+	    			<th>School/College Name</th>
+	    			<th>Address</th>
+	    			<th>Duration</th>
+	    		</tr>
+	    	</thead>
+	    	<tbody>
+	    		<tr>
+	    			<th>ABSS Institute of Technology</th>
+	    			<td>Salarpur, Mawana road, Meerut, Uttar Ptadesh, 250001</td>
+	    			<td>Aug 2015 - June 2019</td>
+	    		</tr>
+	    		<tr>
+	    			<th>SBSS College</th>
+	    			<td>Begusarai, Bihar, 250001</td>
+	    			<td>May 2013 - March 2015</td>
+	    		</tr>
+	    		<tr>
+	    			<th>Smt RS High School</th>
+	    			<td>Hanspur, Begusarai, Bihar, 851126</td>
+	    			<td>May 2011 - March 2013</td>
+	    		</tr>
+	    		<tr>
+	    			<th>UMS Saidpur</th>
+	    			<td>Saidpur, Begusarai, Bihar, 851129</td>
+	    			<td>Apr 2003 - Apr 2011</td>
+	    		</tr>
+	    	</tbody>
+	    </table>
+	    <br>	

```

Comment:
No need to use a `<br />`. There are times to use it, but that's mostly within flow content (like `<p>`)